est/001.est

```diff
@@ -0,0 +1,22 @@
+date: 28-02-2016
+author: Dali Freire
+method: champions.pert
+scope:
+  1: Infrastructure setup
+  2: Design the operation commands
+  3: GitHub/Issue trackers integration
+  4: Project site
+  5: Integrate with continuous integration systems
```

Comment:
@original-brownbear I'm talking about [this](http://www.yegor256.com/2015/03/29/rultor-with-appveyor.html) and the [Continuous Integration](https://github.com/yegor256/rultor) section of the readme. But, yes, I think you are right, maybe it's not in the scope of the Rultor project.